sdk/python/feast/repo_operations.py

```diff
@@ -167,33 +160,43 @@ def apply_total(repo_config: RepoConfig, repo_path: Path):
         if registry_view.name not in repo_table_names:
             views_to_delete.append(registry_view)
 
+    sys.dont_write_bytecode = False
+    for entity in repo.entities:
+        registry.apply_entity(entity, project=project, commit=False)
+        click.echo(
+            f"Registered entity {Style.BRIGHT + Fore.GREEN}{entity.name}{Style.RESET_ALL}"
+        )
+
```

Comment:
Is there a reason why this was moved to after the feature views?

Reply:
Yes, this is necessary because calling `list_feature_tables` and `list_feature_views` will forcibly refresh the cache, causing any uncommitted changes to be lost, so we either need to (a) immediately commit the `apply_entity` changes or (b) call `apply_entity` after `list_feature_tables` and `list_feature_views`.

Reply:
ok cool